### ISSUE TITLE
Mass Empty Mails send

### ISSUE THREAD

**Issue by bppushly** [freescout-helpdesk/freescout#2428]
We have futher problems with using freescount and exchange.

A customer just wrote me that he got an empty mail every two mintues,
i went to logs and see this:

Screenshot: https://bit.ly/3VBmbwJ
It is the same customer again and again. I censored the mailadress for privacy.

This is a desaster. How could this happend and what can we do to avoid it?

**Comment by freescout-helpdesk** [comment]
Check logs. Are you using any workflows?

**Comment by bppushly** [comment]
No workflows. Plugin not even installed.

The send mail error log shows nothing for today.

What could be special about this ticket is that it was in "all" mailbox and i assigned it to a agent.

**Comment by freescout-helpdesk** [comment]
Check Manage > Logs > App Logs

**Comment by bppushly** [comment]
I disconnected the exchange account now. 

[2022-11-29 17:36:02] Fetching finished
1	[2022-11-29 17:36:02] Fetching UNREAD emails for the last 3 days.


But no logs for the time where all the mails have been sent

**Comment by bppushly** [comment]
Jut rechecked the sent mail logs, it started the massmailing at 11 in the morning and stopped at 16 when i clicked "disconnect" on exchange connection.

Also after i disconnected it received two new tickets that were made two hours ago and sent an autoreply.

**Comment by bppushly** [comment]
I think I got the problem.

Here is the error on server

[2022-11-29 12:32:22] production.ERROR: Could not save outgoing reply to the IMAP folder, IMAP folder not found: Gesendete Elemente -  Error: Undefined offset: 1; File: /home/pushly/ticket/vendor/webklex/php-imap/src/Connection/Protocols/ImapProtocol.php (138)  

I entered an "IMAP"-Folder for sending and called it Gesendete Elemente.

But the strange thing is, why it did work for all other Mails, but not for this one?
